src/settings/AudioSettings.tsx

```diff
@@ -253,25 +232,22 @@ export default function GeneralSettings(props: ISettingsPanelProps) {
           </IconButton>
         </Tooltip>
       </Box>
-      <Box component="h3" sx={headingStyle}>
-        Output Settings
-      </Box>
       <Box component="span" sx={{ display: 'flex', alignItems: 'center' }}>
-        <FormControl sx={{ my: 1 }}>
-          <InputLabel id="demo-simple-select-label" sx={style}>
+        <FormControl sx={formControlStyle}>
+          <InputLabel id="ouput-label" sx={selectStyle}>
             Output Devices
           </InputLabel>
           <Select
             name="audioOutputDevices"
-            labelId="select-audio-devices"
+            labelId="output-label"
             id="audio-output-device"
             multiple
             value={output}
-            label="Output"
+            label="Output Devices"
+            sx={selectStyle}
             onChange={(e) => {
               handleMultiSelect(DeviceType.OUTPUT, e);
             }}
-            input={<OutlinedInput label="Tag" />}
```

Comment:
I believe this was the culprit for the visual bug with the borders shown above. Not entirely sure why but I'm guessing it injects an html element somewhere that interferes with some css rules MUI uses for the select field
![image](https://user-images.githubusercontent.com/13799608/215664219-da724ac6-dac9-4d6e-89f0-9f4cb63a98ad.png)